### PR TITLE
NEW ADMIN: jk788@nau.edu

### PR DESCRIPTION
**Pull request by jakereps**

### CODE REVIEW

File: keyrings/live/blackbox-admins.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ebolyen@gmail.com
 jai.rideout@gmail.com
+jk788@nau.edu
+KEYNAME
```
Comment (thermokarst) [inline keyrings/live/blackbox-admins.txt]: Whats going on here with `KEYNAME`?

Comment (jakereps) [inline keyrings/live/blackbox-admins.txt]: Oops, it apparently added the auto ran copy-paste.

Comment (jakereps) [inline keyrings/live/blackbox-admins.txt]: Fixed

**Comment by jakereps** [comment]
Now it should be good. Apparently even though it erred out from auto-running a copied over instruction:

```shell
Develop@myMac-2 secrets (master) $ blackbox_addadmin KEYNAME
gpg: WARNING: nothing exported
gpg: keyring '/Users/Develop/Developer/work/secrets/keyrings/live/secring.gpg' created
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
```

 it still added the `KEYNAME` to the file...

**Pull request merged**